sdks/python/container/base_image_requirements_manual.txt

```diff
@@ -25,9 +25,7 @@
 # Consider constraining requirements of Beam itself when necessary.
 
 bs4  # Commonly used HTML processing tool.
-# Don't upgrade to Cython 3.x, until it's released, stable and we have consensus
-# to upgrade. Use 0.xx for now.
-cython<1
+cython<4
```

Comment:
- are the changes here still compatible with Cython0.x, or shall we use cython>=3,<4 here?

Reply:
The code change that needs to happen for cython 3.0.0 to be happy is still compatible with 0.29.x so we should be good, but I'm definitely in favor of going with the fixed bound instead of < 4.